lib/hooks/gitlab/templates/pipeline.html.haml

```diff
@@ -0,0 +1,36 @@
+:ruby
+  label_classes = {
+    'failed'  => 'label-danger',
+    'created' => 'label-info',
+    'manual'  => 'label-info',
+    'running' => 'label-info',
+    'success' => 'label-success',
+    'success_with_warnings' => 'label-warning'
+  }
+  web_url = payload.project.web_url
+  p_status = payload.object_attributes.status
+  p_label_class = label_classes[p_status].presence || 'label-default'
+  p_id = payload.object_attributes.id
+  p_url = "#{web_url}/pipelines/#{p_id}"
+
+%div.col-xs-12
```

Comment:
@masumiya Could we remove class names that start with `.col-` in this template?
I did not realize that these grid-layout does not work well on smaller devices :sweat_smile: 

![selection_304](https://user-images.githubusercontent.com/43346/53714654-9526b500-3e92-11e9-947f-671e85388199.png)


Besides, since the length of each status name is not the same, fixing the width of these badges seems difficult (e,g., `success_with_warnings` is 3 times as long as `success`).

Reply:
orz..
I replaced  from `col-xs-*` to `col-lg-*`, and remove the `&nbsp;` in `<li>*</li>`, `success_with_warnings ` is changed `success(warnings)`. 
Thank you so much !!